engine-tests-connector/src/utils.rs

```diff
@@ -336,6 +502,19 @@ pub fn validate_eth_address(address: &str) -> Address {
     Address::decode(address).unwrap()
 }
 
+pub fn get_aurora_contract() -> Vec<u8> {
+    let file = "../bin/aurora-engine-mainnet.wasm";
```

Comment:
```suggestion
    let file = "../bin/aurora-mainnet.wasm";
```

Reply:
we can't use
`    let file = "../bin/aurora-mainnet.wasm";`

This name conflicts with other builds